appveyor.yml

```diff
@@ -10,6 +10,7 @@ install:
   - npm install
 
 test_script:
+  - npm run clean
```

Comment:
Not sure if this is necessary assuming CI creates a new workspace for each build.

Reply:
> assuming CI creates a new workspace for each build

That's probably not true, the failure in 1st commit implies the app scaffolding test run twice without cleaning the first created app, see:
https://github.com/strongloop/loopback-next/pull/1509/commits
Therefore the app version bumps to a new patch.

Reply:
let me check the doc of appveyor see how it works...

Reply:
hmm take back my word, the cli package doesn't need a `dist*` folder at all.